test/e2e/e2e.go

```diff
@@ -159,6 +161,71 @@ func WithChannelAndSubscriptionReady(clients *test.Clients, channel *v1alpha1.Ch
 	return nil
 }
 
+// CreateBroker will create a Broker
```

Comment:
Trailing period.

Reply:
Done. Sorry about all the trailings. It was just copy paste from other methods. None had so I was lazy and maintained "consistency" :)